src/main/java/com/ait/lienzo/client/widget/DragMouseControl.java

```diff
@@ -0,0 +1,25 @@
+package com.ait.lienzo.client.widget;
+
+public enum DragMouseControl {
```

Comment:
Not sure if this enum type is expected to be implemented as an `EnumWithValue` in lienzo-core, because it's on the `widgets` package, what you think @SprocketNYC ?